src/main/kotlin/no/nav/syfo/identhendelse/IdenthendelseService.kt

```diff
@@ -43,13 +48,17 @@ class IdenthendelseService(
     }
 
     // Erfaringer fra andre team tilsier at vi burde dobbeltsjekke at ting har blitt oppdatert i PDL før vi gjør endringer
-    private fun checkThatPdlIsUpdated(nyIdent: PersonIdent) {
+    private fun checkThatPdlIsUpdated(nyIdent: PersonIdent, oldIdentUuid: UUID): Boolean {
+        var isUpdated = true
         runBlocking {
             val pdlIdenter = pdlClient.hentIdenter(nyIdent.value) ?: throw RuntimeException("Fant ingen identer fra PDL")
             if (nyIdent.value != pdlIdenter.aktivIdent && pdlIdenter.identhendelseIsNotHistorisk(nyIdent.value)) {
-                throw IllegalStateException("Ny ident er ikke aktiv ident i PDL")
+                log.warn("Identhendelse: Could not update ident with uuid $oldIdentUuid, skipping identhendelse")
+                isUpdated = false
```

Comment:
Går også an å returnere herfra (`return@runBlocking false`) og fra en `else` i stedet for egen variabel

Reply:
Ja, så at IntelliJ foreslo det, var bare ikke sikker på om det var et kjent mønster for oss og om dette ble mer leselig? 🤔